.github/workflows/ci.yml

```diff
@@ -0,0 +1,85 @@
+---
+name: OpenWISP Users CI Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Python==${{ matrix.python-version }} | ${{ matrix.django-version }}
+    runs-on: ubuntu-20.04
+
+    services:
+      redis:
+        image: redis
+        ports:
+          - 6379:6379
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
+        django-version:
+          - django~=2.2
+          - django~=3.0
+          - django~=3.1
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install python dependencies
+        run: |
+          pip install -U pip wheel
+          pip install ${{ matrix.django-version }}
+          pip install -U -r requirements-test.txt
+      - name: Install npm dependencies
+        run: sudo npm install -g jshint
+
+      - name: Install openwisp-users
+        run: pip install -e .[rest]
+
+      - name: QA checks
+        run: |
+          ./run-qa-checks
+
+      - name: Tests
+        run: |
+          coverage run --source=openwisp_users runtests.py
+          SAMPLE_APP=1 coverage run --append --source=openwisp_users runtests.py
+          # basic tests without allauth.socialaccount
+          NO_SOCIAL_APP=1 ./tests/manage.py test testapp.tests.test_admin.TestUsersAdmin --parallel
+
+      - name: Upload Coverage
+        run: coveralls
```

Comment:
I think this should be coveralls `coveralls --service=github` as in https://github.com/openwisp/openwisp-monitoring/pull/262/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR82